### ISSUE TITLE
Cannot distinguish editor in output pane from the other editors

### ISSUE THREAD

**Issue by egamma** [Microsoft/vscode#2368]
Testing #2145 

Select text in the output window.

NVDA reports the current focus (NVDA+Tab) as

>Editor content  edit  focused  multi line

You cannot distingush the output pane from a content editor.

**Comment by bpasero** [comment]
@alexandrudima I guess we need some kind of editor option to pass in so that I can control this?

**Comment by alexandrudima** [comment]
:+1: I'll add an editor option

**Comment by alexandrudima** [comment]
@bpasero Just added an option: `"ariaLabel"`. Can you please adopt it in the workbench editors?

Here is the message I get in Visual Studio: 
![image](https://cloud.githubusercontent.com/assets/5047891/12646432/6113686e-c5d0-11e5-8251-14c75f1f99d8.png)

I would not go with such a long message. Maybe [editor name|file name] + 'text editor'?

**Comment by bpasero** [comment]
I am not sure we should announce the file name as part of the editor, we should clarify this is what others do (why is VS not?) and then we can add it. For now I added the following labels:
* readonly editor (e.g. a file on the index): Readonly text editor
* writeable editor for untitled file: Untitled file text editor
* readonly diff editor: Readonly text diff editor
* writeable diff editor: Text diff editor
* writeable file: Text file editor
* output panel: Output panel

**Comment by alexandrudima** [comment]
@bpasero The reason would be because in VS you can actually tab to the "tab name" which reads you the file name. For us, you have no clue what file name you are trying to edit.

verifier: please check if you can hear the file name you are tabbing into for editing in any way

**Comment by bpasero** [comment]
Makes sense, we can do this for Feb 2016, currently the file name will not be read at all.

**Comment by bpasero** [comment]
https://github.com/Microsoft/vscode/issues/2526

**Comment by alexandrudima** [comment]
:+1:
